centos.org/jobs/foreman-pipelines.yml

```diff
@@ -118,3 +118,17 @@
     type:
       - foreman
       - plugins
+
+- project:
+    name: foreman-duffy
+    jobs:
+      - 'foreman-pipeline-{type}-{flavor}-{version}'
+    empty: ''
+    flavor:
+      - deb
+      - rpm
+    version:
+      - nightly
+    type:
+      - foreman
+      - plugins
```

Comment:
we could, for consistency, also use the `flavor` template for katello/luna/pulpcore? and hardcode flavor to `rpm`.

thoughts?

Reply:
I'd prefer that. That way we don't have jobs defined that will never run.